src/CBT.NuGet.AggregatePackage/build/After.CBT.NuGet.props

```diff
@@ -18,6 +18,7 @@
   <Import Project="$(CBTAggregatePackageImport)" Condition=" '$(CBTAggregatePackageImport)'!='' and '$(CBTEnableAggregatePackageRestore)'=='true' "/>
 
   <PropertyGroup Condition=" '$(CBTEnableAggregatePackageRestore)' == 'true' "> 
+    <CBTNuGetAggregatePackagePropertyFile Condition=" '$(CBTNuGetAggregatePackagePropertyFile)' == '' And '$(IntermediateOutputPath)' == '' ">$(MSBuildProjectDirectory)\obj\AggregatePackages.props</CBTNuGetAggregatePackagePropertyFile> 
```

Comment:
Isn't there a better solution than this?  Is `IntermediateOutputPath` set too late?  Maybe this should go into `CBTNuGetIntermediateOutputPath`?

Reply:
sure